activesupport/lib/active_support/claims.rb

```diff
@@ -0,0 +1,57 @@
+require 'active_support/time'
+
+module ActiveSupport
+  class Claims
```

Comment:
It's actually showing up in the documentations :smile: I will add a ```:nodoc:``` there then.